src/test/java/api/loans/RenewalAPITests.java

```diff
@@ -787,6 +789,28 @@ public void cannotRenewWhenItemIsDeclaredLost() {
       hasUUIDParameter("itemId", smallAngryPlanet.getId()))));
   }
 
+  @Test
+  public void cannotRenewWhenItemIsClaimedReturned() {
+    final IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource jessica = usersFixture.jessica();
+    final String comment = "testing";
+    final DateTime dateTime = DateTime.now();
+
+    final JsonObject loanJson = loansFixture.checkOutByBarcode(smallAngryPlanet, usersFixture.jessica())
+      .getJson();
+
+    loansFixture.claimItemReturned(new ClaimItemReturnedRequestBuilder()
+        .forLoan(loanJson.getString("id"))
+        .withItemClaimedReturnedDate(dateTime)
+        .withComment(comment));
+
+    final Response response = attemptRenewal(smallAngryPlanet, jessica);
+
+    assertThat(response.getJson(), hasErrorWith(allOf(
+      hasMessage(CLAIMED_RETURNED_RENEWED_ERROR),
```

Comment:
I don't think the API tests should share the production definition of the error message, as this makes it more likely they will succeed when the error message is incorrect.